src/packages/v4/views/V4ProjectDashboard/hooks/useV4ProjectHeader.ts

```diff
@@ -24,20 +28,37 @@ export interface ProjectHeaderData {
 export const useV4ProjectHeader = (): ProjectHeaderData => {
   const router = useRouter()
   const { projectMetadata, projectId } = useProjectMetadataContext()
-  // const {
-  //   handle,
-  //   projectOwnerAddress,
-  //   totalVolume,
-  //   trendingVolume,
-  //   paymentsCount,
-  //   createdAt,
-  // } = useContext(V2V3ProjectContext) -> How it looks on v2v3
+  
+  const _projectId = BigInt(projectId ?? 0)
+
+  const { data: projectOwnerAddress } = useReadJbProjectsOwnerOf({
+    args: [_projectId],
+  })
+
+  const { data } = useSubgraphQuery(
+    ProjectsDocument,
+    {
+      where: {
+        projectId,
+      },
+    },
+  )
+
+  const projectStatsData = data?.projects?.[0]
+  const {
+    createdAt,
+    volume: totalVolumeStr,
+    trendingVolume: trendingVolumeStr,
+    paymentsCount,
+  } = projectStatsData ?? {
```

Comment:
Hmm, should we have fallbacks like this? Do we need them?

Reply:
I just wasn't sure how best to handle `projectStatsData` possibly being undefined otherwise?